### PR TITLE
The great 9 armor massacre of 2023-2024(And red and black fixer nerfs)

### PR DESCRIPTION
**Pull request by CrabbytheCrab**
## About The Pull Request

Nerfs all 3 aleph armors that have 9 in one of their stats.

Mimicry has 8/5/5/6(I DID IT KIRIE I TOUCHED MIMICRY!!!)
Smile has 5/5/8/6
Flesh is Willing has 8/6/6/4

Also does these changes to red and black fixer
Black Fixer:
Melee Damage -10 (Upper and Lower, it has Rapid Melee 2 so it still swings for a good chunk) (New Range 30-40, Rapid Melee 2, 60-80 from 80-100)
Hammer Damage -50 (200 from 250)
Pulse Damage -2 (effective -10 total damage nerf)

Red Fixer:
Melee Damage Lower -15
Melee Damage Upper -5 (New Range 15-35, Rapid Melee 4, 60-140 from 120-160)
Multislash Damage -15 (60 x2, from 75 x2)
Beam Stun Time +1 Second (Sleep Check Death from 0.5 Seconds to 1.5, deadly ability but stops him from chasing *and* lets you get in on him)
## Why It's Good For The Game

9 red and black was honestly overpowered in my eyes for 3 reasons:
1. Red and Black are the 2 most common damage types.
2. having 90% damage reduction made many threats way too easy unless you balanced late game around those but then those threats would deal way too much damage to people without 90% dr.
3. Kod's blessing make them 95% which practically made you immune and allow for silly shit like face tanking claw with mimicry scythe.

This isn't as much a nerf to the abnormalities that have it since even with out 9s on the armor Nothing There and Mosb would still be some of the most picked alephs due to easy of work and their weapons being some of the best in the game.(It might make Til the Last Shot less picked due to buffing agent stats can cause no-one to have the stats to work it and that with it not having 9 armor really makes it less useful to have.).

Does not change Flowering nights even though it has pale 9 due to it being both aleph+, it bein the second most expensive item an abnormality has, only one person can work on rose at a time, people using rose's pe for the 100 pe armor and weapon, pale being the rarest damage type to be fighting against, and it coming with a downside of black 3. If anything maybe make kod blessing quadruple pale taken when with pale 9 since that's the only real reason why pale 9 can be broke at times. Also doesn't change De capo since white immunity is balanced by the fact white can't kill you unlike red, black, and pale. Does not also change the red 9 of nagel und hammer grosshammer and the pale 9 of Jacket of Blue since both are from gacha and I don't want to piss off Kirie from changing those stats.

Kirie wanted me to nerf fixers also and lance gave me what I should nerf about them
## Changelog
:cl:
balance: rebalanced Mimicry, Smile, and Flesh is Willing, nerfs red and black fixer also
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Speaking with maintainers.

**Comment by CrabbytheCrab** [comment]
Good

**Review by EgorDinamit — DISMISSED** [review]
Looks good. I see no problems with it, but I'd wait until someone else also approves of it.

**Review by LanceSmites328 — APPROVED** [review]

**Comment by Kitsunemitsu** [comment]
Okay, spoke with maintainers. 
Speak with Coxswain and Redacted about nerfing White Dusk to about the same power as White Midnight.

This PR is good to merge on 14/1/24, 2 weeks from now, and will be Tmed from time to time before that.

**Comment by CrabbytheCrab** [comment]
is this ready to be merged yet or no?

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**
